### DISCUSSION TITLE
Intercepting radials

### DISCUSSION THREAD

**Discussion by foxmike80** [flybywiresim/aircraft#2798] [Help]
Hello,

I've been trying to do some old-fashioned raw-data navigation and want to follow a radial inbound a VOR. When I select an intercept heading in AP HDG mode and activate the LOC, the PFD shows HDG in green and LOC in blue, but the aircraft immediately turns to the programmed flight plan course. 

In the picture I have manually selected HDG 234 to intercept the 150 radial inbound a station. That station is behind me on my flight plan - I'm turning around. When I activate LOC I would expect the plane to hold heading 234 to intercept the radial and then turn *right* to follow the radial. What actually happens is that the plane turns *left* to follow the original flight plan route.
However, once I have manually intercepted the radial and then select LOC it follows the radial.
Is it me doing something wrong or is there something wrong with the plane?
Thanks for any support,

Matt
![A320LOC](https://user-images.githubusercontent.com/76132268/103315579-3f87a900-4a26-11eb-8edf-c09fda14b2d0.png)

**Comment by foxmike80** [comment]
Ok, did some research and it seems the modern A320s cannot intercept radials. Would not have expected that.  I guess this topic can be closed then.

**Comment by robinottawa** [comment]
Im not surprised if that's true. VOR nav is for little guys, no?

**Reply by derl30n** [reply]
well VOR navigation is fun, but the a320 is able to do that, the real one at least.

**Comment by CaptVirtual** [comment]
Isn't this procedure achieved in airliners via the FMGC/FMC by creating a new waypoint entering the VOR id and the radial/distance  
e.g.  ABC300/50             ABC300/-50 will place a waypoint 50nm inbound from ABC on the 300 radial.

I believe you can also do it via   Direct To Page - Radial In       
[http://a320dp.com/A320_DP/nav-flight-management/sys-14.9.0.html](url)

**Reply by derl30n** [reply]
yes you can do that, you can create waypoint and even can program and radial intercept. You can also do this manually without any flight plan programming by tuning the rad nav page.
